wallet-new/src/Cardano/Wallet/Kernel/DB/Spec/Pending.hs

```diff
@@ -7,7 +7,7 @@
 -- > import           Cardano.Wallet.Kernel.DB.Spec.Pending (Pending)
 -- > import qualified Cardano.Wallet.Kernel.DB.Spec.Pending as Pending
 module Cardano.Wallet.Kernel.DB.Spec.Pending (
-    Pending -- opaque
+    Pending (..) -- visible only to enable delta compression until we find a better solution.
```

Comment:
Uhm, this is not something to be done lightly. I totally understand you needed access to internal fields, but perhaps we can find a better way? Maybe some lenses?

Reply:
I think the idea of opaque is that others modules don`t know the Underlying types and this module can change it. Compression must know it however and it must have access to the whole Map. Even with lenses or some wrapper function won`t it be basically equivalent?